spec/javascript/components/molecules/ContentCardGrid.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { render } from 'enzyme'
+import ContentCardGrid from '~/components/molecules/ContentCardGrid'
+
+describe('ContentCardGrid', () => {
+  test('it renders a content card grid with two content cards', () => {
+    const wrapper = render(
+      <ContentCardGrid
+        title='Content card grid title'
+        description='Content card grid description'
+      />
+    )
+    expect(wrapper.find('.content-card')).toHaveLength(2)
```

Comment:
instead of using render here you can just check if a `ContentCard` component was rendered.

```
    import ContentCard from ''

    ...
    const wrapper = shallow(
      <ContentCardGrid
        title='Content card grid title'
        description='Content card grid description'
      />
    )
    expect(wrapper.find(ContentCard)).toHaveLength(2)
```

That's a little better for a couple reasons:
1. it's more unit-y, the test won't break if the classname in ContentCard changes
2. you're able to use the ContentCard type directly instead of a string selector.

Reply:
yeah that makes sense 👍